src/device-config.ts

```diff
@@ -298,18 +298,8 @@ export async function getCurrent(): Promise<Dictionary<string>> {
 	return currentConf;
 }
 
-export async function formatConfigKeys(
-	conf: Dictionary<string>,
-): Promise<Dictionary<any>> {
-	const backends = await getConfigBackends();
-	const formattedKeys: Dictionary<any> = {};
-	for (const backend of backends) {
-		_.assign(
-			formattedKeys,
-			configUtils.formatConfigKeys(backend, validKeys, conf),
-		);
-	}
-	return formattedKeys;
+export function formatConfigKeys(conf: Dictionary<string>): Dictionary<any> {
+	return configUtils.formatConfigKeys(validKeys, conf);
```

Comment:
Just call this function directly from the caller of `formatConfigKeys`, no need to have this wrapper function.

Reply:
I thought about this as well but `configUtils.formatConfigKeys` takes 2 parameters instead of one like device-config's version. That first parameter (validKeys) is a value set in device-config and once I began changing the code to directly call configUtils function it required moving validKeys to configUtils and it just began to feel wrong. Having this function in device-config adds extra information that config utilities needs.

Reply:
great suggestion @codewithcheese! I just added this change and it really cleaned stuff up.